exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/AggPruleBase.java

```diff
@@ -61,11 +61,9 @@ protected AggPruleBase(RelOptRuleOperand operand, String description) {
   // currently won't generate a 2 phase plan.
   protected boolean create2PhasePlan(RelOptRuleCall call, DrillAggregateRel aggregate) {
     PlannerSettings settings = PrelUtil.getPlannerSettings(call.getPlanner());
-    if ( settings.isForce2phaseAggr() ) { // for testing - force 2 phase aggr
-      return true;
-    }
     RelNode child = call.rel(0).getInputs().get(0);
-    boolean smallInput = child.getRows() < settings.getSliceTarget();
+    boolean smallInput = child.getRows() < settings.getSliceTarget() &&
```

Comment:
Overloading smallInput makes it confusing.  Why not modify the if condition below to : 
    (... || (smallInput && !settings.isForce2phaseAggr()) 
This way the meaning of smallInput is not changed.